practitioners/README.md

```diff
@@ -39,7 +40,11 @@ The FHIR `Practitioner` and `PractitionerRole` resources are mapped from the HL7
                     "Dr"
                 ]
             }
-        ]
+        ],
+        "identifier": [ {
+          "system": "https://fhir.hl7.org.uk/Id/gmp-number",
```

Comment:
Is the system field fixed to "https://fhir.hl7.org.uk/Id/gmp-number", do we need to add this to the table?

Reply:
the link has been added to the table